src/views/Orders.vue

```diff
@@ -274,6 +280,26 @@ export default defineComponent({
         await this.store.dispatch('webhook/unsubscribeWebhook', { webhookId: webhook?.id, shopifyConfigId: this.shopifyConfigId })
       }
     },
+    async updateAutoCancelDays(){
+      if(this.autoCancelDays != this.updatedAutoCancelDays){
```

Comment:
I think we should check if selected product store is not the None while setting and getting

Reply:
Made the requested change.